CONTRIBUTING.md

```diff
@@ -0,0 +1,103 @@
+# Contributing to Tombolo Digital Connector
+
+Thank you for choosing to contribute to this project. **We need your superpowers!**  
+To learn more about the Tombolo vision checkout our [website](http://www.tombolo.org.uk/).
+
+Before starting to contribute have a look at the [README](README.md) and [docs](documentation). Our suggestion is to 
+start with the [system architecture](documentation/System-Architecture.md) and [local datastore](documentation/Local-Datastore.md), and 
+eventually go further in the documentation with other topics of your interest.
+
+## Help wanted
+[Open Source Community - Digital Connector](https://github.com/FutureCitiesCatapult/TomboloDigitalConnector/milestone/22)
+milestone contains issues we think are interesting for you to work on. You can find them also through 
+[`help wanted`](https://github.com/FutureCitiesCatapult/TomboloDigitalConnector/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) 
+tag. Feel free to check the other milestones in case you find something that excites you more.
+If none of the issues reflects your problem/feature create a new one following the 
+[issue template](https://github.com/FutureCitiesCatapult/TomboloDigitalConnector/blob/master/.github/ISSUE_TEMPLATE.md).
+Include screenshots and error log if available. 
+
+## In progress
+Chosen the issue you want to work on, let us know by tagging one of the project maintainers: *@thanosbnt*, 
+*@sassalley*, *@lorenaqendro*, *@arya-hemanshu* and change the issue tag from `help wanted` to `in progress`. Regular
+ update on its status is much appreciated, so that everyone knows its proceeding and eventually help.
+
+## Choosing an issue and start contributing
+If you are new to github you might want to read more about [issues](https://help.github.com/articles/about-issues/)
+and [pull requests](https://help.github.com/articles/about-pull-requests/). 
+To contribute to the project follow these steps:
+- [Fork the repository](https://help.github.com/articles/fork-a-repo/).
+- [Choose the issue](https://github.com/FutureCitiesCatapult/TomboloDigitalConnector/milestone/22) you want to start
+working on, change its tag to `in progress` and tag a project maintainer to inform about it.
+- Use the Github issue to discuss on the requirements, ask for more information, update on the progress or upload 
+images related to the issue.
+- Create a pull request when you feel you have addressed the issue.
```

Comment:
May be add a link for `pull request` to the docs https://help.github.com/articles/about-pull-requests/